packages/stable/src/api/geospatial/featureAPI.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  BaseResourceAPI,
+  CogniteExternalId,
+  ExternalId,
+} from '@cognite/sdk-core';
+import {
+  Feature,
+  FeatureAggregateParams,
+  FeatureCreateItem,
+  FeatureOutputParams,
+  FeatureSearchFilter,
+  FeatureSearchStreamFilter,
+} from './types';
+
+export class FeatureAPI extends BaseResourceAPI<Feature> {
+  public create = (
+    featureTypeExternalId: CogniteExternalId,
+    features: FeatureCreateItem[]
+  ): Promise<Feature[]> => {
+    return this.createEndpoint<FeatureCreateItem>(
+      features,
+      this.url(`${featureTypeExternalId}/features`)
+    );
+  };
+
+  public retrieve = (
+    featureTypeExternalId: CogniteExternalId,
+    externalIds: ExternalId[],
+    params?: FeatureOutputParams
+  ): Promise<Feature[]> => {
+    return this.callEndpointWithMergeAndTransform(externalIds, request =>
+      this.postInParallelWithAutomaticChunking({
+        path: this.url(`${featureTypeExternalId}/features/byids`),
+        items: request,
+        params,
+      })
+    );
+  };
+
+  public update = (
+    featureTypeExternalId: CogniteExternalId,
+    changes: FeatureCreateItem[]
```

Comment:
Request body for update endpoint is different than for create endpoint. Does it actually work in that form?